md/migrate-from-an-earlier-version-of-bonita-bpm.md

```diff
@@ -170,24 +170,23 @@ The `bonita_home` and the database have been migrated.
 14. *Only before 7.3.0* In your new `bonita-target-version` folder, delete the `bonita` folder. You will use the `bonita_home` that you have migrated instead.
 15. *Only before 7.3.0* Set the bonita_home system property to point to the migrated `bonita_home`.
 16. [Configure the bundle to use the migrated database](database-configuration.md). Do not recreate the database.
-16. <a id="compound-permission-migration"/> In the case where deployed resources have required dedicated [authorizations to use the REST API](resource-management.md#permissions), these authorizations are not automatically migrated.  
-    Some manual operation have to be done on the `client/tenants/[TENANT_ID]/conf/compound-permissions-mapping.properties` file that is either located in the _$BONITA\_HOME_ folder if version <7.3.0 or in the extracted `platform_conf/current` folder in version >=7.3.0 (see [for more information](BonitaBPM_platform_setup.md#update_platform_conf) ). 
-    You need to merge file versions, the previous one and the migrated one.
-        
-    For example, The [REST API Extension](rest-api-extensions.md) _CI Users REST API_ has been deployed and the `client/tenants/[TENANT_ID]/conf/compound-permissions-mapping.properties` has been automatically updated with the permissions of this resource (a new line has been added in the file):
-    ```
-    custompage_ciuserRestExtension=[profile_visualization, tenant_platform_visualization, organization_visualization]
-    ```  
-    You need to copy the line of each resource permission to the migrated `client/tenants/[TENANT_ID]/conf/compound-permissions-mapping.properties` file.
-17. Configure License:
+17. <a id="compound-permission-migration" /> In the case where deployed resources have required dedicated [authorizations to use the REST API](resource-management.md#permissions), these authorizations are not automatically migrated.  
+    Some manual operation have to be done on files that are  located in the _$BONITA\_HOME_ folder if version <7.3.0 or in the extracted `platform_conf/current` folder in version >=7.3.0 (see [for more information](BonitaBPM_platform_setup.md#update_platform_conf) ). You need to merge file versions, the previous one and the migrated one.
```

Comment:
IMHO, the `for more information` link should contain the page title
sentence `You need to merge file versions, the previous one and the migrated one` may be changed to `You need to merge the previous file version and the migrated one`